torch/_dynamo/eval_frame.py

```diff
@@ -1022,14 +1022,6 @@ def graph_with_interpreter(*args):
     )
 
     if (shape_env := getattr(fake_mode, "shape_env", None)) is not None:
-        dim_constraints = shape_env.dim_constraints
-        assert dim_constraints is not None
-        dim_constraints.solve()
-        log.warning(
-            "Summary of dimension constraints:%s",
-            dim_constraints.prettify_results(inspect.signature(f)),
-        )
-
         # Inline constraints added by users correspond to unbacked symbols in shape_env,
```

Comment:
Can we just move these lines for inline constraints up and delete the if branch shape_env?

Reply:
The following lines are for adding "inline_constraints" to new_graph.meta which doesn't happen until after the aten_graph=True call.

Reply:
Then, we probably can move the lines down instead of moving lines up. Can we do that? Another reason is that fake_mode is checked to be not None in L904, we shouldn't do the getattr(fake_mode...) before L904.

Reply:
I'm not sure moving it down is going to work, because we want the message to be attached to any constraint violation error, which is thrown above.

Reply:
yeah, I got your point. I just feel it's kind of redundant to do the checking again. The other alternative is to throw the exception later but it seems to be not so desirable as well..